FhirToDataLake/docs/Deployment.md

```diff
@@ -53,16 +53,21 @@ This solution enables you to query against the entire FHIR data with tools such
     | Data Start | Start time stamp of the data to be exported. |
     | Data End | Start time stamp of the data to be exported. Leave it empty if you want to periodically export data in real time.  |
     | Container name | A name for the Storage Account container to which Parquet files will be written. The Storage Account with autogenerated name will automatically be created during the installation. |
+    | Customized Schema | Whether to enable the customized schema for the pipeline, the ```Customized Schema Image Reference``` need to be provided if this value been set to ```true```. |
+    | Customized Schema Image Reference | The customized schema image reference for the image on Container Registry. Refer [TemplateManagement](https://github.com/microsoft/FHIR-Converter/blob/main/docs/TemplateManagementCLI.md) for how to manage your template images. |
     | Filter Scope (new) | For data filtering use. The export scope can be `System` or `Group`. The default value is `System` if no filter is applied.|
     | Group Id (new) | For data filtering use. If the Filter scope is set as `Group`, you need to fill the group Id, otherwise leave it blank.|
     | Required Types (new) | For data filtering use. Specify which types of resources will be included. For example, _type=Patient_ would return only patient resources. All resource types will be exported if not specified. Leave it blank if no filter is applied.|
     | Type Filters (new) | For data filtering use. Use along with the requiredTypes configuration. The value is a comma-separated list of FHIR queries that further restrict the results. All data of requiredTypes will be exported if not specified. Leave it blank if no filter is applied.|
     | Package url | The build package of the agent. You need not change this. |
-    | App Insight Location | You can find logs in the deployed application insight resource. You need not change this. |
+    | Deploy App Insights | Whether to deploy the Application Insights. You can find logs in the deployed application insight resource. You need not change this. |
+    | App Insight Location | The location to deploy the App Insight. You need not change this. |
```

Comment:
Maybe this setting can be changed to choose a location that has quota for app insights? @QuanWanxx